assets/blocks/src/style.scss

```diff
@@ -0,0 +1,20 @@
+/**
+ * The following styles get applied both on the front of your site
+ * and in the editor.
+ *
+ * Replace them with your own styles or remove the file completely.
+ */
+
+.wp-block-tour-block-tour-block {
+	background-color: #fff;
+	color: #000;
+	padding: 2px;
+}
+
+ul.tour-block {
+	margin: 0.5rem 0;
+	li {
+		padding-left: 1rem;
+		text-decoration: underline;
+	}
+}
```

Comment:
Do we really need this CSS? Wouldn't the theme CSS suffice?

Reply:
Removed.